crates/turborepo-lib/src/client.rs

```diff
@@ -92,26 +96,20 @@ pub struct UserResponse {
 }
 
 pub struct APIClient {
-    token: String,
     client: reqwest::Client,
     base_url: String,
 }
 
-#[async_trait]
-impl UserClient for APIClient {
-    fn set_token(&mut self, token: String) {
-        self.token = token
-    }
-
-    async fn get_user(&self) -> Result<UserResponse> {
+impl APIClient {
```

Comment:
Changed from having a `token` field in `APIClient` because we don't want to require a token to construct `APIClient`, as we need a client to do SSO login